DotaStats/Services/ImageService.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import UIKit
+
+protocol ImageService: AnyObject {
+    func loadWithUrl(_ url: String, _ completion: @escaping (Result<UIImage, HTTPError>) -> Void) -> Cancellable?
```

Comment:
Если написать так, то будет выглядеть более свифтово. Ну и дописывать что конкретно мы пытаемся загрузить тоже хорошо
```suggestion
    func fetchImage(with url: String, completion: @escaping (Result<UIImage, HTTPError>) -> Void) -> Cancellable?
```